### DISCUSSION TITLE
Multi repo project

### DISCUSSION THREAD

**Discussion by vietj** [jreleaser/jreleaser#1694] [Q&A]
Does JReleaser handle projects whose artefacts are distributed among a set of different GitHub repositories ?

**Comment by aalmiray** [comment]
Distributed in which sense? 

1. Single source repo. artifacts are published to different repos (multiple gh releases)
2. Multiple source repos. Artifacts are released to their owning repo. 

Option 2 is supported as long as each repo is released individually. 

Option 1 is not supported (yet) ad the tool creates a release per repository. It can not create multiple releases for multiple repos with a single config file.

**Reply by vietj** [reply]
That would be option 2 in my case, our project (Vert.x) is a set of repositories (with a release order) that are released sequentially I the release order. I'm trying to see if jreleaser is a good fit for releasing the project.

**Reply by aalmiray** [reply]
I see. In this case I'd suggest configuring JReleaser on each releasable project/repository. 

Automating a full release of all repositories would require invoking JReleaser on each project in a particular order. This can be done in several ways. Here are a few ideas:

1. Check out each project at adjacent directories. Use a script that cds into each directory and invoke JReleaser. The script knows the order. It would have to check if the latest release operation was successful before continuing to the next. 

2. Use GH actions to configure a workflow that reacts to release events (or your custom event).  Each project may filter the origin to determine the correct triggering order.
